### DISCUSSION TITLE
Texture mesh with input file

### DISCUSSION THREAD

**Discussion by drperpen** [cdcseacave/openMVS#820] [Q&A]
Hi,
I am trying to use the texture mesh module with a .ply mesh and a set of camera poses and images. I am struggling to find a .txt reference file with the cameras transformation matrix and the images path format that I should use. I am trying this:
```
#  camera number, center x, center y, center z, rotation matrix R00, R01, R02, R11, R12, R22, R20, R21, R22, image_path
```
But I get this error:
```
[App     ] error: invalid project
```
Thanks!

**Comment by cdcseacave** [comment]
This question was answered many times in other issue posts, pls use `Interface.h` file to write your poses, cameras, etc, see wiki https://github.com/cdcseacave/openMVS/wiki/Interface

**Reply by drperpen** [reply]
Before posting, I did several searches for using the texture module as a standalone process with a .txt file input, and I didn't find anything.
